File: server/router/lgjx/testApply.go
package lgjx

import (
	v1 "github.com/flipped-aurora/gin-vue-admin/server/api/v1"
	"github.com/flipped-aurora/gin-vue-admin/server/middleware"
	"github.com/gin-gonic/gin"
)

type TestApplyRouter struct {
}

func (s *TestApplyRouter) InitApplyRouter(Router *gin.RouterGroup) {
	testApplyRouter := Router.Group("testApply").Use(middleware.OperationRecord())
	testApplyRouterWithoutRecord := Router.Group("testApply")
	var testApplyApi = v1.ApiGroupApp.LgjxTestApiGroup.TestApplyApi
	{
		testApplyRouter.POST("createApply", testApplyApi.CreateApply)             // 新建Apply
		testApplyRouter.DELETE("deleteApply", testApplyApi.DeleteApply)           // 删除Apply
		testApplyRouter.DELETE("deleteApplyByIds", testApplyApi.DeleteApplyByIds) // 批量删除Apply
		testApplyRouter.PUT("updateApply", testApplyApi.UpdateApply)              // 更新Apply
	}
	{
		testApplyRouterWithoutRecord.GET("findApply", testApplyApi.FindApply)       // 根据ID获取Apply
		testApplyRouterWithoutRecord.GET("getApplyList", testApplyApi.GetApplyList) // 获取Apply列表
	}
}

File: web/src/api/lgjx/testApply.js
import service from '@/utils/request'

export const createApply = (data) => {
  return service({
    url: '/testApply/createApply',
    method: 'post',
    data
  })
}

export const deleteApply = (data) => {
  return service({
    url: '/testApply/deleteApply',
    method: 'delete',
    data
  })
}

export const deleteApplyByIds = (data) => {
  return service({
    url: '/testApply/deleteApplyByIds',
    method: 'delete',
    data
  })
}

export const updateApply = (data) => {
  return service({
    url: '/testApply/updateApply',
    method: 'put',
    data
  })
}

export const findApply = (params) => {
  return service({
    url: '/testApply/findApply',
    method: 'get',
    params
  })
}

export const getApplyList = (params) => {
  return service({
    url: '/testApply/getApplyList',
    method: 'get',
    params
  })
}
